Add upstream fix in git 9b7d886b.  This should be in
version 3.24.2

diff --git a/gtk/gtktooltip.c b/gtk/gtktooltip.c
index f9156624ed..e5fc5dbdaa 100644
--- gtk/gtktooltip.c
+++ gtk/gtktooltip.c
@@ -897,6 +897,9 @@ gtk_tooltip_position (GtkTooltip *tooltip,
                 "gtk-cursor-theme-size", &cursor_size,
                 NULL);
 
+  if (cursor_size == 0)
+    cursor_size = gdk_display_get_default_cursor_size (display);
+
   if (device)
     anchor_rect_padding = MAX (4, cursor_size - 32);
   else
